src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -85,6 +86,8 @@ export class RbdFormComponent implements OnInit {
     '16 MiB',
     '32 MiB'
   ];
+  action: string;
+  resource = 'rbd';
```

Comment:
I think this should allways be all caps: `RBD`

Reply:
Done.